src/index.html

```diff
@@ -17,11 +17,445 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a
+          href="#"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="logo"
```

Comment:
alt attribute should describe the image content
![image](https://github.com/user-attachments/assets/c7f91b0c-1941-47c1-b187-db5fcd00f84a)